packages/server/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "module": "NodeNext",
-    "moduleResolution": "NodeNext"
+    "moduleResolution": "NodeNext",
+    "lib": ["ESNext"]
```

Comment:
🤔  Do you remember what required this?  Should we add it to the root `tsconfig.json` instead?

I have a soft goal of keeping the per-package `tsconfig.json` files as minimal as possible, to promote unified config as much as possible.

Reply:
There are some APIs that differ between different lib versions, ie. in this case I couldn't use the `cause` field on Error because it requires lib specified to `ES2021` or later

Reply:
We could add it to root as it's probably a sane default since we already have it in `core`